.gitignore

```diff
@@ -1,3 +1,28 @@
-BrewMobile.xcodeproj/xcuserdata/
-BrewMobile.xcworkspace/xcuserdata/
-Carthage/Build
\ No newline at end of file
+## Build generated
+build/
+DerivedData
+
+## Various settings
+*.pbxuser
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
+xcuserdata
+
+## Other
+*.xccheckout
+*.moved-aside
+*.xcuserstate
+*.xcscmblueprint
+
+## Obj-C/Swift specific
+*.hmap
+*.ipa
+
+# Carthage
+Carthage/Checkouts
```

Comment:
it was actually committed [on purpose](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#carthagecheckouts), to make sure it will always be available